### PR TITLE
Updating README links to work with doc changes

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

**Review by cretz — APPROVED** [review]

**Comment by cretz** [comment]
@flossypurse - This good to be merged?

**Comment by flossypurse** [comment]
No yet.
This PR works in conjunction with this PR: https://github.com/temporalio/documentation/pull/813
Because the samples repo README is Snipsynced into the docs repo, these changes need to sit in their own branch until the docs PR lands. 
Probably not the best design any more - so will look at changing that up.

File: README.md [hunk 1]
```diff
@@ -45,13 +43,13 @@ Each sample demonstrates one feature of the SDK, together with tests.
...
 
-- **Sessions**: Demonstrates how to bind a set of Activity Executions to a specific Worker after the first Activity executes. This feature is showcased in the [File Processing example](https://github.com/temporalio/samples-go/tree/master/fileprocessing). Addition documentation: [How to use Sessions in Go](https://docs.temporal.io/docs/go/sessions).
+- **Sessions**: Demonstrates how to bind a set of Activity Executions to a specific Worker after the first Activity executes. This feature is showcased in the [File Processing example](https://github.com/temporalio/samples-go/tree/master/fileprocessing). Addition documentation: [How to use Sessions in Go](https://docs.temporal.io/docs/go/how-to-use-worker-sessions-in-go).
```
Comment (flossypurse) [inline README.md:50]:
```suggestion
- **Sessions**: Demonstrates how to bind a set of Activity Executions to a specific Worker after the first Activity executes. This feature is showcased in the [File Processing example](https://github.com/temporalio/samples-go/tree/master/fileprocessing). Addition documentation: [How to use Sessions in Go](https://docs.temporal.io/docs/go/sessions).
```

File: README.md [hunk 2]
```diff
@@ -45,13 +43,13 @@ Each sample demonstrates one feature of the SDK, together with tests.
 
 - [**Encryption**](https://github.com/temporalio/samples-go/tree/master/encryption): How to use encryption for Workflow/Activity data with the DataConverter API. Also includes an example of stacking encoders (in this case encryption and compression). [Docs](https://docs.temporal.io/docs/go/workflows/#custom-serialization-and-workflow-security).
 
-- [**Query Example**](https://github.com/temporalio/samples-go/tree/master/query): Demonstrates how to Query the state of a single Workflow Execution using the `QueryWorkflow` and `SetQueryHandler` APIs. Additional documentation: [How to Query a Workflow Execution in Go](https://docs.temporal.io/docs/go/queries).
+- [**Query Example**](https://github.com/temporalio/samples-go/tree/master/query): Demonstrates how to Query the state of a single Workflow Execution using the `QueryWorkflow` and `SetQueryHandler` APIs. Additional documentation: [Queries in Go](https://docs.temporal.io/docs/go/queries).
```
Comment (flossypurse) [inline README.md:46]:
```suggestion
- [**Query Example**](https://github.com/temporalio/samples-go/tree/master/query): Demonstrates how to Query the state of a single Workflow Execution using the `QueryWorkflow` and `SetQueryHandler` APIs. Additional documentation: [How to Query a Workflow Execution in Go](https://docs.temporal.io/docs/go/queries).
```

**Pull request merged**